DownloadCollection.user.js

```diff
@@ -4,14 +4,29 @@
 // @version      1.2
 // @description  Opens the download page for each item in your collection.
 // @author       Ryan Bluth, Xerus2000
-// @match        https://bandcamp.com/YOUR_USERNAME
+// @match        https://bandcamp.com/USERNAME
 // @grant        GM_openInTab
 // ==/UserScript==
 
 // Ignore albums with the same title and artist
-var ignoreDuplicateTitles = true
+var ignoreDuplicateTitles = true;
 // The number of milliseconds spent scrolling down to load all albums
-var albumLoadDuration = 10000;
+var albumLoadDuration = 30000; //30000 ms default - 30 sec default
+//If true will make the time it takes to open new links entirely random. If false will use a set interval
+const randomInterval = false; 
```

Comment:
What's the purpose of having this option?

Reply:
Was my initial approach. I left it in because I figured since I already programmed it having the option would be nice